src/CoreBundle/private/library/classes/dbobjects/TCMSTableConf.class.php

```diff
@@ -281,13 +281,13 @@ public function &GetField($sFieldName, &$oTableRow, $loadDefaults = false)
     }
 
     /**
-     * returns an iterator of field definitions that match the types given in aFieldTypes
+     * Returns a record list of field definitions that match the types given in aFieldTypes
      * (aFieldTypes = `constname` in cms_field_type).
      *
      * @param array $aFieldTypes
      * @param bool  $bDoNotUseAutoObjects - set to true if you want to prevent the class from using auto objects
      *
-     * @return TIterator
+     * @return TCMSRecordList|TdbCmsFieldConfList
```

Comment:
TIterator is not wrong, but totally a useless info here, because it is always a recordList and autocomplete is not working with TIterator, so i fixed this.